pkg/shed/field_string.go

```diff
@@ -65,6 +65,9 @@ func (f StringField) Put(val string) (err error) {
 
 // PutInBatch stores a string in a batch that can be
 // saved later in database.
-func (f StringField) PutInBatch(batch *leveldb.Batch, val string) {
-	batch.Put(f.key, []byte(val))
+func (f StringField) PutInBatch(batch *badger.Txn, val string) {
+	err := batch.Set(f.key, []byte(val))
+	if err != nil {
+		f.logger.Debugf("could not set values in PutInBatch")
```

Comment:
The error is not handled, not even put in log message.

This is very dangerous. We should return this error in order to stop any other actions on the batch. This can be cause of data inconstancies.

Reply:
yeah. I will add error..